exonum/src/storage/list_index.rs

```diff
@@ -134,7 +142,7 @@ where
         self.base.get(&index)
     }
 
-    /// Returns the last element of the list, or `None` if it is empty.
+    /// Returns the last element of the list or `None` if it is empty.
```

Comment:
how can the last element be empty? there is always an element that can be considered last at a certain point in time. am I wrong? 
probably, the idea is that the element may have an empty value?